### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by RandomAhri** [nazgulsenpai/sbxigncode#3]
I'm confused on how this works. Could anyone help me use this?

**Comment by nazgulsenpai** [comment]
This is the source code for the custom XignCode binary used in my Scarlet Blade Vendetta launcher for Wine. It contains usage instructions. https://github.com/nazgulsenpai/sbwinelauncher 

Sorry for the super late reply.

**Comment by RandomAhri** [comment]
I've followed the steps, but the game doesn't open up. I've downloaded wine-stable 5.0. After running sb.py, I get:

0034:fixme:virtual:NtQueryVirtualMemory (process=0x1a4,addr=0x7b410000) Unimplemented information class: MemorySectionName
0034:fixme:virtual:NtQueryVirtualMemory (process=0x1a4,addr=0x7bc10000) Unimplemented information class: MemorySectionName
0034:fixme:winsock:set_dont_fragment IP_DONTFRAGMENT for IPv4 not supported in this platform

Any idea what I should do?
